src/main/scala/org/clulab/reach/extern/export/fries/FriesOutput.scala

```diff
@@ -321,205 +323,209 @@ class FriesOutput extends JsonOutputter {
-      val fromId = lookupMentionId(mention, entityMap, eventMap)
-      if (fromId.isDefined) {
-        frames ++= mk1toMLinkFrames(paperId, mention, fromId.get,
-                                    assemblyAPI.getEquivalenceLinks(mention), "equivalent-to",
-                                    passageMap, entityMap, eventMap)
-        frames ++= mk1to1LinkFrames(paperId, mention, fromId.get,
-                                    assemblyAPI.getCausalPredecessors(mention), "preceded-by",
-                                    passageMap, entityMap, eventMap)
-      }
+        throw new RuntimeException("ERROR: unknown argument type in makeArgument: " + arg.labels)
     }
-
-    assemblyModel                           // return the constructed assembly model
+    pm
   }
 
 
-  private def mkContextFrame (paperId:String,
-                              passage:FriesEntry,
-                              mention:BioMention,
-                              contextId:String,
-                              context:ContextMap): PropMap = {
+  /** Create and return a new context frame using the given mention and context info. */
+  private def makeContextFrame (paperId: String,
+                                passage: FriesEntry,
+                                mention: BioMention,
+                                contextId: String,
+                                context: ContextMap): PropMap = {
     val f = startFrame()
     f("frame-type") = "context"
     f("frame-id") = contextId
     f("scope") = mkSentenceId(paperId, passage, mention.sentence)
-    f("facets") = mkContextFacets(context)
+    f("facets") = makeContextFacets(context)
     f
   }
 
-  private def mkContextFacets (context: ContextMap): PropMap = {
-    val m = new PropMap
-    m("object-type") = "facet-set"
+  /** Create and return a new map containing the context facets from the given context info. */
+  private def makeContextFacets (context: ContextMap): PropMap = {
+    val pm = new PropMap
+    pm("object-type") = "facet-set"
     context.foreach { case(k, v) =>
       val friesFacetName = contextNameMap.getOrElse(k, k)
-      m(friesFacetName) = v
+      pm(friesFacetName) = v
     }
-    m
+    pm
   }
 
-  private def mkEntityMention(paperId:String,
-                              passageMeta:FriesEntry,
-                              mention:BioTextBoundMention,
-                              contextIdMap:CtxIDed,
-                              entityMap: IDed): List[PropMap] = {
+
+  /** Return a tuple of maps mapping entities to unique IDs and events to unique IDs. */
+  private def makeEMaps (paperId: String,
+                         mentions: Seq[Mention],
+                         passageMap: Map[String, FriesEntry]): (IDed, IDed) = {
+    val entityMap = new IDed
+    val eventMap = new IDed
+
+    for (mention <- mentions) {
+      mention match {
+        case em:BioTextBoundMention =>
```

Comment:
I feel like labels would be more future-proof (ex. `case entity if entity matches "Entity" =>`), but if it works...